src/dnx/dnx.cpp

```diff
@@ -235,7 +235,11 @@ int CallApplicationProcessMain(int argc, LPTSTR argv[])
     bool fSuccess = true;
     HMODULE m_hHostModule = nullptr;
 #if CORECLR_WIN
-    LPCTSTR pwzHostModuleName = _T("dnx.coreclr.dll");
+#if ONECORE
+        LPCTSTR pwzHostModuleName = _T("dnx_onecore.coreclr.dll");
```

Comment:
I think tabs are good. This is a nested `#if`. VS will automatically move precompiler directives to the left and adding nesting to the actual content signals it is a nested if.